docs/kbs_attestation_protocol.md

```diff
@@ -413,18 +413,15 @@ A POST request with the content of resource to `/kbs/v0/resource/<repository>/<t
 
 ### Token Resource
 
-Authenticated attesters can also request a token from the KBS.
+Authenticated attesters can also request an attestation token from the KBS.
```

Comment:
What we're providing through that endpoint are attestation results, I think:

```suggestion
Authenticated attesters can also request an attestation result token from the KBS.
```

We probably should also update the section title from `Token Resource` to `Attestation Result`.